pkg/minikube/out/out.go

```diff
@@ -87,6 +90,21 @@ func Step(st style.Enum, format string, a ...V) {
 	String(outStyled)
 }
 
+// Step writes a stylized and templated message to stdout
+func SpinnerStep(st style.Enum, format string, a ...V) {
```

Comment:
fix the comment here.

maybe rename this function to
"StepSpinning"

Reply:
done, check it :)